hydromt_fiat/fiat.py

```diff
@@ -313,6 +313,8 @@ def setup_exposure_buildings(
         max_potential_damage: Union[str, Path],
         ground_floor_height: Union[int, float, str, Path, None],
         unit: str,
+        gfh_unit: str = None,
```

Comment:
it would be nicer for the arguments to be better grouped, e.g, gfh_unit should be after ground_floor_height etc.